src/templates/Studio/Wonder/World/World.js

```diff
@@ -9,10 +9,31 @@ const World = ({ filename, layout, onImportWorld, scene }) => {
 
     const importWorld = async () => {
       SceneLoader.ShowLoadingScreen = false;
-      importedModel = await SceneLoader.AppendAsync(filename.path, filename.file, scene);
+
+      importedModel = await SceneLoader.AppendAsync(
+        filename.path,
+        filename.file,
+        scene,
+      );
+      const belt = await scene.meshes[1];
+
       model = scene.meshes.find(mesh => mesh.id === '__root__');
-      console.log(importedModel, model)
-      model.scaling.z = 1;
+
+      const material = new StandardMaterial('CustomSTANDARTMaterial', scene);
+      if (belt) {
+        belt.material = material;
+      }
+
+      // makes belt lighter
+      material.emissiveColor = new Color3(1, 1, 1);
+
+      const texture = new Texture('models/sculpture/checkers_big.png');
```

Comment:
Having trouble finding this file. Did you add it?

Reply:
Yeah I deleted it by mistake previously but re-added it